template.tpl

```diff
@@ -0,0 +1,277 @@
+  },
+  "description": "Chartbeat is a real-time, web analytics product. Our code sends tracking pings from your users\u0027 browsers to our servers and allows you to monitor traffic and engagement in real-time.",
+  "containerContexts": [
+    "WEB"
+  ]
+}
+
+
+___TEMPLATE_PARAMETERS___
+
+[
+  {
+    "type": "LABEL",
+    "name": "Instructions",
+    "displayName": "\u003cb\u003eInstructions:\u003c/b\u003e Fill out the fields below. For further assistance please see our \u003ca href\u003d\"http://support.chartbeat.com/docs/\"\u003eimplementation docs\u003c/a\u003e."
+  },
+  {
+    "type": "TEXT",
+    "name": "uid",
+    "displayName": "uid",
+    "simpleValueType": true,
+    "help": "Your Chartbeat UID (Account ID). See \u003ca href\u003d\"http://support.chartbeat.com/docs/#accounts\"\u003eChartbeat standard configuration parameters\u003c/a\u003e.",
+    "alwaysInSummary": true,
+    "valueValidators": [
+      {
+        "type": "NON_EMPTY"
+      }
+    ]
+  },
+  {
+    "type": "TEXT",
+    "name": "domain",
+    "displayName": "domain",
+    "simpleValueType": true,
+    "valueValidators": [
+      {
+        "type": "NON_EMPTY"
+      }
+    ],
+    "help": "Domain parameter. See \u003ca href\u003d\"http://support.chartbeat.com/docs/#code\"\u003eChartbeat standard configuration parameters\u003c/a\u003e.",
+    "alwaysInSummary": true
+  },
+  {
+    "type": "TEXT",
+    "name": "sections",
+    "displayName": "sections",
+    "simpleValueType": true,
+    "help": "Comma-separated list of sections. See \u003ca href\u003d\"http://support.chartbeat.com/docs/#sectionsauthors\"\u003eChartbeat standard configuration parameters\u003c/a\u003e and \u003ca href\u003d\"https://www.google.com/url?q\u003dhttps://support.google.com/tagmanager/answer/6164391?hl%3Den\u0026sa\u003dD\u0026ust\u003d1593065281792000\u0026usg\u003dAFQjCNGIkL6gneEEH0OkcfEw18Nx1z3D7A\"\u003eData Layers\u003c/a\u003e."
+  },
+  {
+    "type": "TEXT",
+    "name": "authors",
+    "displayName": "authors",
+    "simpleValueType": true,
+    "help": "Comma-separated list of authors. See \u003ca href\u003d\"http://support.chartbeat.com/docs/#sectionsauthors\"\u003eChartbeat standard configuration parameters\u003c/a\u003e and \u003ca href\u003d\"https://www.google.com/url?q\u003dhttps://support.google.com/tagmanager/answer/6164391?hl%3Den\u0026sa\u003dD\u0026ust\u003d1593065281792000\u0026usg\u003dAFQjCNGIkL6gneEEH0OkcfEw18Nx1z3D7A\"\u003eData Layers\u003c/a\u003e."
+  },
+  {
+    "type": "CHECKBOX",
+    "name": "flickerControl",
+    "checkboxText": "Enable Headline Testing Flicker Control?",
+    "simpleValueType": true,
+    "help": "By default this is set to false. If you are using our Headline Testing tool and need to implement Flicker Control, please review our \u003ca href\u003d\"http://support.chartbeat.com/docs/headlinetesting.html\"\u003edocumentation\u003c/a\u003e before enabling this feature."
+  }
+]
+
+
+___SANDBOXED_JS_FOR_WEB_TEMPLATE___
```

Comment:
We should have a FE person review the below to make sure it's idiomatic JS. Want to ping Dolores on it?